### PR TITLE
Edge Device Groups introduction blog post

### PR DESCRIPTION
**Pull request by jakub-dzon**
Signed-off-by: Jakub Dzon <jdzon@redhat.com>

### CODE REVIEW

File: _posts/2022-04-08-introducing-device-groups.markdown [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+tags:
+  - guide
+  - flotta
+  
+summary-1: Introduction to edge device groups
+---
+
+If you are Flotta user that needs to manage large numbers of devices and doesn't want to change configuration of each 
+of them separately, using Edge Device Groups is the perfect approach. With the use of Edge Device Groups you can change 
+configuration of any number of device by changing ony one CR - `EdgeDeviceGroup` and this blog post will show you how.
```
Comment (gciavarrini) [inline _posts/2022-04-08-introducing-device-groups.markdown:16]: s/ony/only/

File: _posts/2022-04-08-introducing-device-groups.markdown [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+
+### EdgeDeviceGroup configuration
+
+User may define the same configuration elements using `EdgeDeviceGroup` as one would with `EdgeDevice` and configuration 
+defined in the `EdgeDeviceGroup` would be applied to all edge devices using it. `EdgeDeviceGroup`  **always** takes 
+precedence over whatever is defined in the `EdgeDevice`as a whole. It means that even if some element is not present in
```
Comment (gciavarrini) [inline _posts/2022-04-08-introducing-device-groups.markdown:22]: I think there is no space between "EdgeDevice" and "as"

**Review by gciavarrini — CHANGES_REQUESTED** [review]

File: _posts/2022-04-08-introducing-device-groups.markdown (hunk 1 above)
Comment (jakub-dzon) [reply _posts/2022-04-08-introducing-device-groups.markdown]: Done.

File: _posts/2022-04-08-introducing-device-groups.markdown (hunk 2 above)
Comment (jakub-dzon) [reply _posts/2022-04-08-introducing-device-groups.markdown]: Indeed. Done.

**Review by gciavarrini — APPROVED** [review]

**Review by machacekondra — APPROVED** [review]

File: _posts/2022-04-08-introducing-device-groups.markdown [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+categories: flotta
+author: Jakub Dżon
+tags:
+  - guide
+  - flotta
+  
+summary-1: Introduction to edge device groups
+---
+
+If you are Flotta user that needs to manage large numbers of devices and doesn't want to change configuration of each
```
Comment (tupyy) [inline _posts/2022-04-08-introducing-device-groups.markdown:14]: s/that/who

File: _posts/2022-04-08-introducing-device-groups.markdown [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+device-2   22s
+device-3   51s
+```
+
+Let's configure `device-1` heartbeat interval to 15 seconds:
+```shell
+kubectl patch edgedevice device-1 --type merge -p '{"spec":{"heartbeat": {"periodSeconds": 15}}}'
+```
+
+which will result in on-device configuration file (`/etc/yggdrasil/device/device-config.json`) to look like this:
```
[the remark this answers is not in the document]
Comment (jakub-dzon) [reply _posts/2022-04-08-introducing-device-groups.markdown:102]: The `on-device` is supposed to underline that it is an effective configuration that is saved on device FS and used there - in contrast to configuration described by CRs. 
I will work on wording in that sentence to be clearer.

[the remark this answers is not in the document]
Comment (jakub-dzon) [reply _posts/2022-04-08-introducing-device-groups.markdown]: Done

**Review by tupyy — APPROVED** [review]

**Review by eloycoto — APPROVED** [review]

**Pull request merged**